src/Uno.UI/UI/Xaml/Controls/WebView/WebView.iOS.cs

```diff
@@ -287,11 +287,6 @@ internal void OnNavigationFailed(WebViewNavigationFailedEventArgs args)
 			NavigationFailed?.Invoke(this, args);
 		}
 
-		public static bool MustUseWebKitWebView()
```

Comment:
Removing this API will create a binary breaking change that is not necessary. Set it to return `true` always, and mark it as `Obsolete` with this PR's url.